docs/Tutorial_Matrix_Profile_Top_Ten_short.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>Can you please provide a description of how to interpret the plots? What is the green subsequence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>